CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
+ - Validate duplicate interface names
```

Comment:
IMHO this should be something like "Warn early when the same interface name is mentioned multiple times" to focus on the user-visible behavior change.

Reply:
Done, thanks!